### PR TITLE
Capstone Project

### PR DESCRIPTION
**Pull request by LionRouge1**
In this PR I :

- Created the 2 pages for mobile.
- Deployed the project.
- Adapted the 2 pages to desktop.
- Implemented the user interaction: link, mobile menu.
- Implemented the section "Featured speakers" with dynamic HTML.
- Deployed the project and checked to make sure everything works as planned.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Global School Conferance 
+
```
Comment (elyor-doniyorov) [inline README.md:4]: - [ ] Please record a 5-minute video presenting the features of the project. Make sure in your video :
 - The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
- The video contains a demo of the project, showing the pages in different screen sizes.
- The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title>Global school about page</title>
+ <script src="https://kit.fontawesome.com/84967187a9.js" crossorigin="anonymous"></script>
+ <link rel="stylesheet" href="css/mobile_style.css">
+ <link rel="stylesheet" href="css/Destop_style.css">
+ <link rel="stylesheet" href="css/about_style.css">
+</head>
+<body>
+ <header>
```
Comment (elyor-doniyorov) [inline about.html]: - [ ] In both desktop size pages make sure 
1. It is divided into a top dark bar and a bottom white bar.
2. The top short bar section with social media links, logout, and language links aligned to the right.
3. The taller bottom bar with a logo and a list of page links.
4. The top bar contains a logo to the left.
1. The top bar contains a list of page links aligned to the right.
Take a look at the original design below 👇 
Screenshot (179)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+     <h3>Lecture</h3>
+     <p>Listen to the speakers from various countries talk about there journey intotech.</p>
+    </div>
+    <div class="program">
+     <img src="images/ignite.png" alt="lecture's icon">
+     <h3>Lecture</h3>
+     <p>Listen to the speakers from various countries talk about there journey intotech.</p>
+    </div>
+   </div>
+   <button type="button">JOIN OUR MEETING</button>
```
Comment (elyor-doniyorov) [inline index.html:80]: - [ ] If we look at the given design we notice that your `JOIN OUR MEETING` button should not have a red background. Let's take a look at the design and fix the design difference 👇 
- Original 👇 
Screenshot (181)
- Yours 👇 
Screenshot (180)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+   </div>
+  </section>
+  <section class="snd_section">
+   <h2 class="Mpr">Main Program</h2><span class="dash"></span>
+   <div class="contain_program">
+    <div class="program">
+     <img src="images/lecture.png" alt="lecture's icon">
+     <h3>Lecture</h3>
+     <p>Listen to the speakers from various countries talk about there journey intotech.</p>
+    </div>
```
Comment (elyor-doniyorov) [inline index.html]: - [ ] For this `Main Program` section please make sure you stick the design as much as possible by using the given `colors` #272A31, #EC5242, #D3D3D3; `font-family`: cocogoose and lato;
Take a look again below images 👇 
- Original design 👇 
Screenshot (182)

- Your design 👇 
Screenshot (183)

File: index.html [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+  </section>
+  <section class="trd_session">
+   <h2>Featured Speakers</h2>
+   <article class="contain_speaker">
+   <!-- Speakers card-->
+
+   </article>
+   <button id="more" type="button">More <i class="fas fa-caret-down"></i></button>
+   <article class="partner">
+    <h2>Partner</h2>
```
Comment (elyor-doniyorov) [inline index.html:90]: - [ ] For your Partner section I recommend again you stick the design by using the given colors and fonts which I mentioned in the previous comment above.

File: about.html [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+ </header>
+ <main>
+  <section class="first_sec">
+   <div>
+    <p class="wel">"Welcome to GLOBAL SCHOOL"</p>
+    <h1 class="title">LET CONTRIBUTE TO THE DEVELOPPEMENT OF OUR COMMUNITY</h1>
+    <p class="intro">The Global School brings together the community of experts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two.
+     This year, we hope to expand our invitation list including organizations and individuals.
+    </p>
+    <p class="hrs">Please contact us for any further questions about GLOBAL SCHOOL</p>
```
Comment (elyor-doniyorov) [inline about.html:49]: - [ ] You have this line 39 sentence is `bold`, while in the given design it is not bold. I recommend you follow the design guidelines.

File: about.html [hunk 7]
```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
```
Comment (elyor-doniyorov) [inline about.html:1]: - [ ] The same thing for the `about `page, kindly use the given `colors(\#272A31, #EC5242, #D3D3D3)` and `fonts(Cocogoose, Lato)` for every section.

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @LionRouge1, 

Good job so far!
## To Highlight 👏 
- [ ] linters are green ✔️ 
- [ ] the app looks awesome 👍 
- [ ] the app is responsive 👍 

There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @LionRouge1 👋🏻, 

Good job making the requested changes 💪🏻!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional Changes ♻️ 

- [ ] In the mobile version of the home page, the footer does not appear as seen in the image below 👇🏻. Please kindly include it.

image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**